feat(ui): Add quality selection UI for audio-only content

nbcl opened the pull request:
## Context

This is a draft pull request that is currently under development.

## Description

I approached the issue by introducing the quality selection to the currently existing resolution button in the UI, conditionally, when audio-only content is being displayed.

```js
tracks = tracks.filter((track, idx) => {
  // If audio-only content is being displayed
  if (this.player.isAudioOnly()) {
    // Keep the first one with the same audioBandwidth.
    otherIdx = tracks.findIndex(
        (t) => t.audioBandwidth == track.audioBandwidth);
  // If video content is being displayed
  } else {
    // Keep the first one with the same height.
    otherIdx = tracks.findIndex(
        (t) => t.height == track.height);
  }
  return otherIdx == idx;
});
```

Currently, this replaces the resolution menu with a quality menu when audio-only content is displayed, as shown in the following image:

<img align="center" src="https://nbcl.github.io/images/bits.png" />

## Draft status

Currently, the following points must be decided before resolving the draft.

- [x] Make sure the obtaining of the audio quality attribute is correct.
- [ ] Decide if a new button must be created to only handle audio quality or having the resolution button shift functionality depending on content is the correct design.

joeyparrish commented on ui/resolution_selection.js:
```diff
@@ -198,14 +208,26 @@ shaka.ui.ResolutionSelection = class extends shaka.ui.SettingsMenu {
-    this.backSpan.textContent =
+    // Assign text attributes depending on content type.
+    if (this.player.isAudioOnly()) {
+      this.button.setAttribute(shaka.ui.Constants.ARIA_LABEL,
```
Looks like you could set a local variable with the localization ID, instead of duplicating all the setAttribute and resolve calls in each branch.

nbcl replied:
Of course, that works way better - Updated it on my last commit! 

```js
// Assign text attributes depending on content type.
const locId = this.player.isAudioOnly() ? LocIds.QUALITY : LocIds.RESOLUTION;
```

nbcl committed:
feat(ui): Add quality selection UI for audio-only content

For audio-only content, in the UI, the resolution button becomes a quality button where users can select their desired audioBandwidth.

Issue: #2071

nbcl committed:
fix(ui): Replace attribute setting duplication with conditional const

Replaces "if" statement that duplicated setAttribute and textContent declarations with a const defined by a conditional operator.

nbcl committed:
fix(ui): Switch from audioBandwidth to bandwidth in quality selection

Switches to bandwidth from audioBandwith in the audio quality selection version of the resolution button to prevent cases in which audioBandwith may not be present.
